### PR TITLE
initial commit of gluster collection

### PR DESCRIPTION
**Pull request by kesavanvt**
Added initial code for gluster collection

### CODE REVIEW

File: plugins/modules/_gluster_heal_facts.py [hunk 1]
```diff
@@ -0,0 +1 @@
+gluster_heal_info.py
```
Comment (pkesavap) [inline plugins/modules/_gluster_heal_facts.py:1]: is this file needed?

**Comment by kesavanvt** [comment]
@pkesavap

File: plugins/modules/_gluster_heal_facts.py (hunk 1 above)
Comment (kesavanvt) [reply plugins/modules/_gluster_heal_facts.py]: _gluster_heal_facts.py was used to achieve the backward compatibility, as the gluster_heal_facts is renamed to gluster_heal_info. 
1. The user can still access the gluster_heal_facts module, however it is a symbolic link to gluster_heal_info. 
2. When gluster_heal_facts is provided as the module name , the ansible core library prefix the module name with '_' and accesses the _gluster_heal_facts module

However with the introduction of collections, it is required to mention module names in fully qualified name and the above process may not be relevant. Thus the _gluster_heal_facts is not needed and has been removed

**Comment by pkesavap** [comment]
Looks Good to Me,
